applications/stepswitch/src/stepswitch_util.erl

```diff
@@ -243,10 +241,10 @@ set_endpoint_caller_id(Endpoint, Name, Number) ->
 
 -spec format_endpoint(wh_json:object(), api_binary(), filter_fun(), api_binary(), wh_json:object(), ne_binary()) ->
                              wh_json:object().
-format_endpoint(Endpoint, Number, FilterFun, DefaultRealm, SIPHeaders, AccountId) ->
+format_endpoint(Endpoint, Number, FilterFun, OffnetReq, SIPHeaders, AccountId) ->
```

Comment:
the change of `DefaultRealm` to `OffnetReq` needs a
change in spec from `api_binary()` to `wapi_offnet_resource:req()`